client/my-sites/pages/page/index.js

```diff
@@ -736,6 +738,15 @@ class Page extends Component {
 		this.props.recordEvent( 'Clicked Delete Page' );
 	};
 
+	recordPageTitleClick = ( canEdit ) => {
```

Comment:
Suggestion: `clickPageTitle()` ? So that it's consistent with the rest of the "callback"s in this file (`editPage()`, `copyPage()` etc)

Reply:
I see you